src/wallet/coincontrol.h

```diff
@@ -93,6 +93,8 @@ class CCoinControl
     FlatSigningProvider m_external_provider;
     //! Locktime
     std::optional<uint32_t> m_locktime;
+    //! Version
+    std::optional<uint32_t> m_version;
```

Comment:
in c8544398fb945b2402274981b22c70fdd8a67b9e

Same comment as for `locktime`

Reply:
With miniscript, it does matter, as it controls whether something with `OP_CSV` is spendable.